PyriteCore/src/world/camera.h

```diff
@@ -12,12 +12,13 @@ namespace pyr
 struct PerspectiveProjection
 {
   //float fovy   = PI * .4f;
-   float fovy = 1.02974f;
+  float fovy = 1.02974f;
```

Comment:
where does that value come from btw?

Reply:
i guess 1.02 * 16/9 ~ 1.81 rad => 100° so you have a fov_x of 100 ?
this is your code, or perhaps erwan's !